### PR TITLE
edited fields in popup, added renderChange to config file

### PR DESCRIPTION
**Pull request by eneedham**
edited fields in popup, added renderChange to config file
also added paragraph to learn more modal

### CODE REVIEW

File: app/assets/js/src/main.js [hunk 1]
```diff
@@ -436,6 +441,20 @@ $(document).ready(function() {
         return (variable + '_y15');
     }
 
+    function y06Var(variable) {
+        if (variable.endsWith('_c')) {
```
Comment (mmcfarland) [inline app/assets/js/src/main.js]: I'm not sure if it's required, but IE does not support `endsWith`, though Edge does.  If it's problematic, you can use `_.last(variable) === '_c'` to achieve the same result.

File: app/assets/js/src/tcviz-config.js [hunk 2]
```diff
@@ -18,133 +18,155 @@ TCVIZ.Config = {
...
         id: 'Statewide Gas Prices'
     }],
     MSA_layers: [{
         text: 'Population Density (gross change)',
         value: 'pp_dn_c',
         absoluteValue: 'pop_dens',
         render: 'number',
-        id: 'Population Density'
+        renderChange: 'number',
+        id: 'Population Density (residents/mile^2)'
```
Comment (mmcfarland) [inline app/assets/js/src/tcviz-config.js]: You can make this a real superscript by changing it to the following:

`'Population Density (residents/mile<sup>2</sup>)'`

screenshot 2017-11-10 at 2 44 38 pm

**Review by mmcfarland — APPROVED** [review]
Looks good and working.  Left a comment about IE and a pointer about rendering superscript, neither of which are deal breakers for me.

**Comment by mmcfarland** [comment]
This looks good and should be working across browsers.  We left a console.log in there for debugging, which out to be removed but doesn't affect the functionality.

**Comment by eneedham** [comment]
just pushed a commit to remove console.log

**Pull request merged**